README.md

```diff
@@ -100,17 +100,36 @@ jobs:
 ```
 
 Some use cases might find the [find-comment](https://github.com/peter-evans/find-comment) action useful.
-This will search an issue or pull request for the first comment containing a specified string, and/or by a specified author.
+This will search an issue or pull request for the first comment containing a specified string, and/or by a specified author, e.g. allowing to update a previously created comment.
 See the repository for detailed usage.
 
 ```yml
-      - name: Find Comment
-        uses: peter-evans/find-comment@v1
-        id: fc
-        with:
-          issue-number: 1
-          comment-author: peter-evans
-          body-includes: search string 1
+    - name: Find Comment
+      uses: peter-evans/find-comment@v1
+      id: fc
+      with:
+        issue-number: ${{ github.event.pull_request.number }} #e.g. 1
+        # comment-author: peter-evans (does not work with "github-actions" author) 
```

Comment:
```suggestion
        comment-author: 'github-actions[bot]'
```

Reply:
Uhh, does it work this way? I will try it out today.

Reply:
Yes, here is an example from one of my workflows.
https://github.com/peter-evans/create-pull-request/blob/master/.github/workflows/ci.yml#L86-L108